### PR TITLE
Fix index page numbering error

### LINKED ISSUE #9: Index numbers are off if there is a ToC
Index numbers generated (i.e., last few pages) are off by 2 if the ToC spans 2 pages.
This is because those numbers don't take into account the 2 ToC pages, which are not numbered.

### PR DESCRIPTION
**Pull request by joel-lim**
Fixes #9 
Fix index page numbering error by accounting for the length of the generated table of contents

### CODE REVIEW

**Comment by damithc** [comment]
@joel-lim thanks for the PR. Ready for review/merging?

**Comment by joel-lim** [comment]
Yup!

File: enhance_textbook/main.py [hunk 1]
```diff
@@ -55,7 +55,12 @@ def add_bookmarks(doc, header_to_pagenumber, headers_and_subs, no_content_pages,
     doc.save(OUTPUT)
 
 
-def generate_index_entries(doc):
+def generate_index_entries(doc, page_offset=0):
+    """
+    Generates a mapping of index terms to lists of page numbers which contain those terms.
```
Comment (damithc) [inline enhance_textbook/main.py:60]: I  think the python convention for the overview statement of a comment is `Generate` not `Generates` (different from Java)

**Comment by damithc** [comment]
Seems to be fixed now. Thanks @joel-lim

**Pull request merged**